backend/package.json

```diff
@@ -13,7 +13,10 @@
       "../.git/HEAD",
       "../git/refs/heads/master",
       "../frontend/lib",
-      "../package.json"
+      "../package.json",
+
+      "../node_modules/swagger-ui-dist/swagger-ui.css",
+      "./node_modules/swagger-ui-dist/swagger-ui.css"
```

Comment:
Why are there 2 swagger-ui.css?

Reply:
Since the move to the monorepo, I've observed two different `node_modules` locations depending on where the npm install was executed. This way, it will be included from whereever that directory decides to pop up